### DISCUSSION TITLE
courses/beginners-web-handbook

### DISCUSSION THREAD

**Discussion by giscus** [crwdu/lms-contents-js101#14] [Q&A]
# courses/beginners-web-handbook

바닐라코딩

https://www.vcdr.club/courses/beginners-web-handbook

**Comment by Ken123777** [comment]
안녕하세요. 바닐라코딩의 켄입니다. 🙇🏻

현재 보시는 콘텐츠의 **각 페이지 하단**에는 질문을 남길 수 있는 댓글창이 있습니다. 깃허브로 로그인하여 댓글 남겨주시면 제가 번개같이 나타나서 답변 남겨드리니 언제든지 무엇이든 편히 질문 남겨주세요.

감사합니다. 🙂
